css/components/_table.scss

```diff
@@ -0,0 +1,97 @@
+.qa-tbl {
+  color: $tundora;
+  display: table;
+  position: relative;
+  width: 100%;
+
+  .btn {
+    height: $sqr-btn-small;
+    width: $sqr-btn-small;
+  }
+
+  .qa-tbl-hdr {
+    cursor: default;
+
+    >div {
+      font-weight: bold;
+    }
+
+  }
+
+  .qa-tbl-row {
+    background-color: $gallery;
+    cursor: pointer;
+    display: table-row;
+    height: $row-height;
+
+    >div {
+      display: table-cell;
+      vertical-align: middle;
+    }
+
+    &:nth-of-type( odd ) {
+      background-color: $alto;
+    }
+
+    .qa-page-id {
+      width: 70px;
+    }
+
+    .qa-page-q-count {
+      width: 120px;
+    }
+
+    .qa-page-preview {
+      width: 74px;
+    }
+
+    &.page:hover {
+      background-color: $alto-dark;
+    }
+
+  }
+
+}
+
+.questions {
```

Comment:
This component seems (but I might be wrong) unrelated to the one above. Should it be in its own file?